zenoh_ros_type/autoware_auto_msgs/autoware_auto_vehicle_msgs.py

```diff
@@ -1,75 +0,0 @@
-from dataclasses import dataclass
```

Comment:
Should we still need this?

Reply:
autoware_auto_vehicle_msgs has been restored. However, FMS does not use these messages, so I didn't add these messages in the newly added autoware_msgs